src/plugins/mer/merbuildsettingswidget.h

```diff
@@ -0,0 +1,55 @@
+/****************************************************************************
+**
+** Copyright (C) 2012 - 2019 Jolla Ltd.
+** Contact: http://jolla.com/
+**
+** This file is part of Qt Creator.
+**
+** GNU Lesser General Public License Usage
+** Alternatively, this file may be used under the terms of the GNU Lesser
+** General Public License version 2.1 as published by the Free Software
+** Foundation and appearing in the file LICENSE.LGPL included in the
+** packaging of this file.  Please review the following information to
+** ensure the GNU Lesser General Public License version 2.1 requirements
+** will be met: http://www.gnu.org/licenses/old-licenses/lgpl-2.1.html.
+**
+** Other Usage
+**
+** Alternatively, this file may be used in accordance with the terms and
+** conditions contained in a signed written agreement between you and Digia.
+**
+****************************************************************************/
+
+#pragma once
+
+#include <projectexplorer/namedwidget.h>
+
+namespace Utils {
+class DetailsWidget;
+}
+
+
+namespace Mer {
+namespace Internal {
+namespace Ui {
+    class MerBuildSettingsWidget;
+}
+
+class MerBuildSettingsWidget : public ProjectExplorer::NamedWidget
```

Comment:
The name of this class does not align with it's use case. MerBuildConfigurationWidget would be better. I would also prefer implementing this class inside merbuildconfiguration.cpp instead of introducing more source files.

Reply:
Related files removed.